core/modules/layout/layout.module

```diff
@@ -2178,7 +2317,7 @@ function layout_flexible_template_load($flexible_template_name = NULL) {
     return $all_templates[$flexible_template_name];
   }
   else {
-    return FALSE;
+    return "";
```

Comment:
Not sure why the return value change here. Returning an empty string isn't consistent with the docblock either. But as above, I'd recommend leaving the return value as-is and update the docblock instead. `@return LayoutFlexibleTemplate|false`